pages/common/fc-cache.md

```diff
@@ -7,10 +7,26 @@
 
 `fc-cache`
 
+- Generate font cache files verbosely:
+
+`fc-cache {{[-v|--verbose]}}`
+
 - Force a rebuild of all font cache files, without checking if cache is up-to-date:
 
 `fc-cache {{[-f|--force]}}`
 
 - Erase font cache files, then generate new font cache files:
 
 `fc-cache {{[-r|--really-force]}}`
+
+- Scan a specific directory:
+
+`fc-cache {{path/to/directory}}`
+
+- Scan system-wide directories (skip user's home directory):
```

Comment:
```suggestion
- Scan system-wide directories, skipping the user's home directory:
```
Parentheses seem a little out of place, especially given the state of the -f command.